index.js

```diff
@@ -18,7 +20,17 @@ github.authenticate({
     token: process.env.FORKED_TOKEN,
 })
 
-const packageJson = shell.cat('package.json')
+const args = process.argv.slice(2)
+if (args.indexOf('-h') !== -1 || args.indexOf('--help') !== -1) {
+  console.log(`  Easy-peasy usage:
+    1. Navigate to directory of dependency you want to fork.
+    2. run \`fork\`
+
+  Lemon-squeezy usage:
+    1. From your project’s root directory, run \`fork dependencyName\`.`)
```

Comment:
I like this, but I think the api would feel a little better if I could say `$ fork node_modules/dependencyName`, that way if for some reason I have a repo with like, a directory of projects I can still use this to fork them. It might also make the logic a little easier to code.